Grid/parallelIO/IldgIO.h

```diff
@@ -649,29 +654,57 @@ class IldgWriter : public ScidacWriter {
 
     stats Stats;
     Stats(Umu,header);
-    
-    std::string format = header.floating_point;
+   
     header.ensemble_id    = description;
     header.ensemble_label = description;
     header.sequence_number = sequence;
     header.ildg_lfn = LFN;
-
-    assert ( (format == std::string("IEEE32BIG"))  
-           ||(format == std::string("IEEE64BIG")) );
-
     //////////////////////////////////////////////////////
     // Fill ILDG header data struct
     //////////////////////////////////////////////////////
     ildgFormat ildgfmt ;
-    const std::string stNC = std::to_string( Nc ) ;
-    ildgfmt.field          = std::string("su"+stNC+"gauge");
+    const std::string stNC = std::to_string(Nc) ;
+    //std::string format;
```

Comment:
As previous comments, keep line 666 as it was, and don't introduce a commented-out code line.